js/ui/viewSelector.js

```diff
@@ -65,6 +65,11 @@ const ViewsDisplayLayout = new Lang.Class({
 
         this._heightAboveEntry = 0;
         this.searchResultsTween = 0;
+        this._compositeMode = false;
```

Comment:
Although this change is motivated by the composite mode, it's actually something to be applied on low enough resolutions, regardless of whether we're using composite mode or not, so I think I would rename this variable (and similar references in the .css file) to prevent confusion :). Something like `this._lowResolutionMode` maybe?